# Request (empty)
---
# Response
uint8 buttons
